src/server/memory/memory.cc

```diff
@@ -341,7 +341,8 @@ Status BulkStoreBase<ID, P>::Delete_GPU(ID const& object_id){
   if (object->arena_fd == -1) {
     auto buff_size = object->data_size;
     GPUBulkAllocator::Free(object->pointer, buff_size);
-    //DVLOG(10) << "after free: " << IDToString(object_id) << ": " << Footprint()
+    // DVLOG(10) << "after free: " << IDToString(object_id) << ": " <<
+    // Footprint()
     //          << "(" << FootprintLimit() << ")";
```

Comment:
ditto

Reply:
Thanks for code review. The above comments have been solved now.